Index: src/main/java/com/kaleyra/academy/sudoku/utils/Preferences.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/kaleyra/academy/sudoku/utils/Preferences.java	(revision de5c1d160cfe443b500ae555779d3d15ac30dc0a)
+++ src/main/java/com/kaleyra/academy/sudoku/utils/Preferences.java	(date 1585240092654)
@@ -120,7 +120,7 @@
     }
 
     private String getFilename() {
-        String path = OSUtils.getInstance().getStrategy().getPropertiesPath();
+        String path = System.getProperty("user.home");
 
         if(path== null || path.isEmpty())
             return PROPERTIES_FILE_NAME;
Index: src/main/java/com/kaleyra/academy/sudoku/model/strategy/generation/GameStrategyFactory.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/kaleyra/academy/sudoku/model/strategy/generation/GameStrategyFactory.java	(revision de5c1d160cfe443b500ae555779d3d15ac30dc0a)
+++ src/main/java/com/kaleyra/academy/sudoku/model/strategy/generation/GameStrategyFactory.java	(date 1585240092673)
@@ -28,6 +28,8 @@
 //        String sudokuValidationStrategy = Preferences.getInstance().getProperty("SudokuValidationStrategy");
 
 
+        Preferences.getInstance().getProperty("mycustom.property");
+
         System.setProperty("com.kaleyra.academy.sudoku.model.NewGameStrategyManager.useFile", "true");
 
         String useFileString =
